tensorflow_recommenders_addons/dynamic_embedding/python/ops/hkv_hashtable_ops.py

```diff
@@ -79,6 +79,7 @@ def __init__(
       evict_strategy=HkvEvictStrategy.LRU,
       step_per_epoch=0,
       gen_scores_fn=None,
+      reserved_key_start_bit=0,
```

Comment:
A simple explanation of config reserved_key_start_bit in document docs/api_docs/tfra/dynamic_embedding/HkvHashTableConfig.md, that would be better.